cmd/kube-apiserver/app/server.go

```diff
@@ -274,7 +278,7 @@ func (s *APIServer) Run(_ []string) error {
 		Authenticator:          authenticator,
 		Authorizer:             authorizer,
 		AdmissionControl:       admissionController,
-		EnableV1Beta3:          v1beta3,
+		DisableV1Beta3:         disableV1beta3,
```

Comment:
Not bad - it's an easy compile error during rebase.  The harder is when it isn't a compile error (no one notices).

----- Original Message -----
> > @@ -274,7 +278,7 @@ func (s *APIServer) Run(_ []string) error {
> >  		Authenticator:          authenticator,
> >  		Authorizer:             authorizer,
> >  		AdmissionControl:       admissionController,
> > -		EnableV1Beta3:          v1beta3,
> > +		DisableV1Beta3:         disableV1beta3,
> 
> @smarterclayton How much of a problem are these types of breaking changes?
> 
> https://github.com/openshift/origin/blob/master/pkg/cmd/server/kubernetes/master.go#L65
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/6098/files#r27528113